package.json

```diff
@@ -48,7 +48,7 @@
   "dependencies": {
     "bluebird": "^3.4.7",
     "bluebird-retry": "^0.10.1",
-    "bmapflash": "^1.2.0",
+    "bmapflash": "^1.2.1",
```

Comment:
Should the `bmapflash` update be mentioned in the commit message (as that version makes it Node v4 compatible)?